examples/angular/angular.json

```diff
@@ -50,7 +51,7 @@
                 {
                   "type": "initial",
                   "maximumWarning": "2mb",
-                  "maximumError": "6mb"
+                  "maximumError": "9mb"
```

Comment:
What's the reason for that change? Was it breaking earlier?

Reply:
We are now adding 2 web-components files - one for API and one for StoplightProject.

Shouldn't be an issue I think? I don't believe people will use *both* of those at the same time? :thinking:

Reply:
Looks like it's about bundle size. I guess we exceeded limit with new package added.